qcodes/instrument_drivers/QuantumDesign/DynaCoolPPMS/DynaCool.py

```diff
@@ -91,30 +93,43 @@ def __init__(self, name: str,
 
         self.add_parameter('field',
                            label='Field strength',
-                           unit='A/m',
+                           unit='T',
+                           get_cmd=self._deprecated_field_getter,
+                           snapshot_value=False)
+
+        self.add_parameter('field_measured',
+                           label='Field',
+                           unit='T',
                            get_cmd=self._present_field_getter)
 
+        self.add_parameter('field_target',
+                           label='Field target',
+                           unit='T',
+                           get_cmd=None,
+                           set_cmd=None,
+                           vals=vals.Numbers(-14, 14))
+
         self.add_parameter('field_setpoint',
                            label='Field setpoint',
-                           unit='A/m',
+                           unit='T',
```

Comment:
that's backwards incompatible, but i guess that's the point anyway, right?

Reply:
This is actually a bugfix, see the original commit: https://github.com/QCoDeS/Qcodes/commit/c9a8bf99d18abc12bb63dab1548973ee990739d7

We've been working in Teslas all along, we just forgot to update the units. So I would not consider this to be backwards incompatible.

Reply:
ah, i see, great ))